docs/docs/en/user-guide/runtimes/autogen/using_non_openai_models.md

```diff
@@ -1,84 +1,85 @@
 # Using Non-OpenAI Models with FastAgency
 
-FastAgency makes it simple to work with **non-OpenAI models** through AutoGen's runtime. You can do this in a couple of ways:
+FastAgency makes it simple to work with **non-OpenAI models** through its [**autogen**](../../../api/fastagency/runtimes/autogen/autogen/AutoGenWorkflows.md) runtime. You can do this in a couple of ways:
```

Comment:
Please search and replace [**autogen**] -> [**AutoGen**]

Reply:
Fixed